src/modules/organisations/organisations.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable, UnprocessableEntityException } from '@nestjs/common';
+import { Repository } from 'typeorm';
+import { Organisation } from './entities/organisations.entity';
+import { OrganisationRequestDto } from './dto/organisation.dto';
+import { InjectRepository } from '@nestjs/typeorm';
+import { User } from '../user/entities/user.entity';
+import { OrganisationMapper } from './mapper/organisation.mapper';
+import { CreateOrganisationMapper } from './mapper/create-organisation.mapper';
+
+@Injectable()
+export class OrganisationsService {
+  constructor(
+    @InjectRepository(Organisation)
+    private readonly organisationRepository: Repository<Organisation>,
+    @InjectRepository(User)
+    private readonly userRepository: Repository<User>
+  ) {}
+
+  async create(createOrganisationDto: OrganisationRequestDto, userId: string) {
+    const emailFound = await this.emailExsits(createOrganisationDto.email);
```

Comment:
It looks like the method name `emailExsits` is misspelt. Did you mean to write: `emailExists`?

Reply:
Yes, it's a typo. I'll fix this right now @coderoyalty